prExplorer/API/obj/project.assets.json

```diff
@@ -1996,11 +1996,11 @@
   "project": {
     "version": "1.0.0",
     "restore": {
-      "projectUniqueName": "c:\\Users\\Ermat\\Desktop\\NC Github\\NC\\prExplorer\\API\\API.csproj",
+      "projectUniqueName": "C:\\Users\\Ermat\\Desktop\\NC Github\\NC\\prExplorer\\API\\API.csproj",
```

Comment:
Do you see why you shouldn't check in those files? They contain a lot of personal info, which should never go to the repository